Dockerfile

```
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1
+FROM caddy:2.10.2-alpine
```

Should we always pull the current latest image?

```suggestion
FROM caddy:alpine
```